amd/samples/runner/timingconfig/shaderarray/builder.go

```diff
@@ -186,10 +186,26 @@ func (b *Builder) connectVectorMem() {
 	for i := range b.numCUs {
 		cu := b.cus[i]
 		rob := b.l1vROBs[i]
-		at := b.l1vATs[i]
 		l1v := b.l1vCaches[i]
 		tlb := b.l1vTLBs[i]
 
+		// Rebuild AddressTranslator with correct configuration
```

Comment:
We should not rebuild the address translator. Please try to set up the right dependency when we build the address translators at the first time. We may need to adjust some building orders.